src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,44 @@
+.nav {
+
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  &__item:not(:first-child) {
+    margin-left: 20px;
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    height: $header-height;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-size: 12px;
+    font-weight: 500;
+    color: $header-link-text-color;
+    transition: color $header-transition-duration;
+
+    /* LoVeHAte */
+    &.is-active,
+    &:hover {
+      color: $header-link-text-hover-color;
+    }
```

Comment:
here you can go something like this (didn't test it):
```scss
&.is-active {
  &:hover {
      color: $header-link-text-hover-color;
  }
}
```

Reply:
now i must duplicate same code for nav__link::hover and nav__link::active. Misha always said "dont repeat yourself".